src/components/Calculator.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
```

Comment:
According to the requirements:

![image](https://user-images.githubusercontent.com/57421823/171382213-563116f4-ebcf-4a6e-a36a-c85db9b4c02d.png)


Your implementation:

![image](https://user-images.githubusercontent.com/57421823/171381969-c4afe78c-3953-4a17-82b1-2f2fdee3be5d.png)


Therefore:

- [ ] Please, match the colors given in the design.
- [ ] Kindly correct the gap in the "0" key.   It should span the 2 columns.